service-api/src/main/java/greencity/dto/tariff/GetTariffInfoForEmployeeDto.java

```diff
@@ -0,0 +1,25 @@
+package greencity.dto.tariff;
+
+import greencity.dto.LocationsDtos;
+import greencity.dto.RegionDto;
+import greencity.dto.courier.CourierTranslationDto;
+import greencity.dto.courier.GetReceivingStationDto;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Optional;
+
+@Data
+@Builder
```

Comment:
Is it required to have setters and builders?

Reply:
done